data/json/items/containers/containers.json

```diff
@@ -4618,6 +4607,27 @@
     },
     "melee_damage": { "bash": 20 }
   },
+  {
+    "id": "nitrox_canister",
+    "type": "TOOL",
+    "category": "tool_magazine",
+    "name": { "str": "nitrox container" },
+    "description": "A small tank of nitrox, a mixture of nitrogen and oxygen used for diving.",
+    "weight": "25 g",
+    "volume": "755 ml",
+    "longest_side": "12 cm",
+    "price": "20 USD",
+    "price_postapoc": "5 USD",
+    "to_hit": { "grip": "none", "length": "hand", "surface": "any", "balance": "uneven" },
+    "material": [ "steel", "plastic" ],
+    "symbol": ";",
+    "color": "green",
+    "pocket_data": [
+      { "pocket_type": "MAGAZINE", "airtight": true, "watertight": true, "rigid": true, "ammo_restriction": { "nitrox": 3 } }
```

Comment:
It needs to be a `Container` type pocket, otherwise `in_its_container` cannot spawn the content with this default container, only the content itself will be spawned.

Reply:
Also, please check other items that may also be affected. With a glance I found out that `extinguishing_agent` and `teargas_spray` are also using `bottle_plastic` as its default container.

Reply:
Yeah I fixed those over an hour ago but the merge conflicts have frozen my github desktop